data/json/items/comestibles/baked.json

```
@@ -37,5 +37,24 @@
     "stack_size": 24,
     "fun": 3,
     "vitamins": [ [ "vitC", 2 ], [ "calcium", 3 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "baklava",
+    "name": { "str_sp": "baklava" },
+    "weight": "50 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 1000,
```

https://www.nutritionix.com/i/nutritionix/baklava-1-piece/546c6fd4a589ba77150f2599
Says 306 calories for a 76g serving, that would put this at about 200 calories, unless im missing something about e.g. stack size.
```suggestion
    "calories": 200,
```